{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unauthorized"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/mgenetwork\/networkerror\/unauthorized"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError\/unauthorized","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Occurs when "},{"type":"codeVoice","code":"401"},{"type":"text","text":" status code is received from the server."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unauthorized"}],"title":"NetworkError.unauthorized","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10MGENetwork12NetworkErrorO12unauthorizedyA2CmF","modules":[{"name":"MGENetwork"}]},"hierarchy":{"paths":[["doc:\/\/MGENetwork\/documentation\/MGENetwork","doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError"]]},"references":{"doc://MGENetwork/documentation/MGENetwork/NetworkError/unauthorized":{"role":"symbol","title":"NetworkError.unauthorized","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"unauthorized"}],"abstract":[{"type":"text","text":"Occurs when "},{"type":"codeVoice","code":"401"},{"type":"text","text":" status code is received from the server."}],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError\/unauthorized","kind":"symbol","type":"topic","url":"\/documentation\/mgenetwork\/networkerror\/unauthorized"},"doc://MGENetwork/documentation/MGENetwork":{"role":"collection","title":"MGENetwork","abstract":[],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork","kind":"symbol","type":"topic","url":"\/documentation\/mgenetwork"},"doc://MGENetwork/documentation/MGENetwork/NetworkError":{"role":"symbol","title":"NetworkError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"NetworkError"}],"abstract":[{"type":"text","text":"Error occurring upon network tasks."}],"identifier":"doc:\/\/MGENetwork\/documentation\/MGENetwork\/NetworkError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"NetworkError"}],"url":"\/documentation\/mgenetwork\/networkerror"}}}